ceph-releases/jewel/ubuntu/14.04/daemon/config.k8s.sh

```diff
@@ -1,9 +1,20 @@
 #!/bin/bash
 set -e
 
+# log arguments with timestamp
+function log {
```

Comment:
Oh and the entrypoint.sh sources all the other entrypoints so we don't need to replicate this function, just keep it in `entrypoint.sh`